03.bowling/bowling.rb

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+scores = ARGV[0].split(',')
+shots = scores.map { |score| score == 'X' ? 10 : score.to_i }
```

Comment:
```suggestion
shots = ARGV[0].split(',').map { |shot| shot == 'X' ? 10 : shot.to_i }
```

あえてローカル変数を使わずに命名の問題を回避する、というアプローチもあります。
（可読性が悪化しないなら）

Reply:
ローカル変数を使わずに命名の問題を回避する、というアプローチを参考に`shots = ARGV[0].split(',').map { |shot| shot == 'X' ? 10 : shot.to_i }`と、ひとまとめにしました。